testsuite/MDAnalysisTests/analysis/test_align.py

```diff
@@ -43,8 +43,8 @@
 warnings.simplefilter('always')
 
 
-class TestRotationMatrix(object):
-    def __init__(self):
+class TestRotationMatrix(TestCase):
+    def setUp(self):
```

Comment:
@utkbansal I assume the goal of this PR is to make it run under pytest and then you will do a second round adapting it to pytest style? Is this correct? (Would be fine with me.)